pkg/matrix-diff/matrix-diff.go

```diff
@@ -0,0 +1,75 @@
+package matrixdiff
+
+import (
+	"fmt"
+
+	"github.com/openshift-kni/commatrix/pkg/types"
+)
+
+// Describes the status of an endpointslice.
+type epsStatus int
+
+// Used = the port is open on the node.
+// NotUsed = the port is not open on the node.
+// Missing = no endpointslice defined for an open port.
+const (
+	used epsStatus = iota
+	notUsed
+	missing
+)
+
+type MatrixDiff struct {
```

Comment:
a []types.ComDetails is basically a comMatrix, so I maybe it's will be better saving it that way, and by that allowing to use it's methods instead of changing the SortAndRemoveDuplicates method to a function for example?

Reply:
agree with shir